src/actions/appActions.js

```diff
@@ -212,8 +217,15 @@ export const initAppAndRedirectAction = () => {
   };
 };
 
-export const setupSentryAction = (user: ?Object, wallet: Object) => {
+export const setupLoggingServicesAction = () => {
   return (dispatch: Dispatch, getState: GetState) => {
+    if (__DEV__) return;
+
+    const {
+      user: { data: user },
+      wallet: { data: wallet },
+    } = getState();
```

Comment:
```suggestion
const user = getState().user.data?.username;
const keyWalletAddress = getState().wallet.data.wallet?.keyWalletAddress;
```

Seems like all of the unwrapping code here and below be simplified here.